{"metadata":{"title":"error","roleHeading":"Instance Property","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"error"},{"kind":"text","text":": "},{"preciseIdentifier":"s:6OpenAI8APIErrorV","kind":"typeIdentifier","text":"APIError"}],"symbolKind":"property","role":"symbol","externalID":"s:6OpenAI16APIErrorResponseV5errorAA0C0Vvp","modules":[{"name":"OpenAI"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"kind":"identifier","text":"error"},{"text":": ","kind":"text"},{"text":"APIError","kind":"typeIdentifier","preciseIdentifier":"s:6OpenAI8APIErrorV","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError"}]}]}],"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openai\/apierrorresponse\/error"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error"},"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse"]]},"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"references":{"doc://OpenAI/documentation/OpenAI":{"url":"\/documentation\/openai","role":"collection","kind":"symbol","type":"topic","title":"OpenAI","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","abstract":[]},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/error":{"title":"error","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"error"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:6OpenAI8APIErrorV","text":"APIError"}],"kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error","url":"\/documentation\/openai\/apierrorresponse\/error","type":"topic","abstract":[],"role":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError":{"fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"APIError","kind":"identifier"}],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","abstract":[],"role":"symbol","type":"topic","navigatorTitle":[{"text":"APIError","kind":"identifier"}],"url":"\/documentation\/openai\/apierror","kind":"symbol","title":"APIError"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse":{"abstract":[],"kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"APIErrorResponse"}],"title":"APIErrorResponse","url":"\/documentation\/openai\/apierrorresponse","fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"text":"APIErrorResponse","kind":"identifier"}],"type":"topic"}}}